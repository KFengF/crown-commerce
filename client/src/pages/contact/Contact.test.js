import React from 'react';
import { shallow } from 'enzyme';
import ContactPage from './Contact.page';

describe('Contact page', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<ContactPage />);
    });

    it('should render Contact page', () => {
        expect(wrapper).toMatchSnapshot();
    });
});