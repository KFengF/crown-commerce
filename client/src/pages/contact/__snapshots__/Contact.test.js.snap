// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact page should render Contact page 1`] = `ShallowWrapper {}`;
